apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/site-logo/index.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * WordPress dependencies
+ */
+import { registerBlockType } from '@wordpress/blocks';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import edit from './edit';
+
+registerBlockType( 'a8c/site-logo', {
+	title: __( 'Site Logo' ),
+	description: __( 'Site Logo' ),
+	icon: 'format-image',
+	category: 'layout',
+	keywords: [ __( 'logo' ), __( 'icon' ), __( 'site' ) ],
+	edit,
+	save: () => null,
```

Comment:
This is the [default behavior](https://github.com/WordPress/gutenberg/blob/a57202e635ff0cd190a09128d27951620db9a870/packages/blocks/src/api/registration.js#L77), so we can remove this line.